src/Samples/Subscriber/Subscriber.csproj

```diff
@@ -6,6 +6,6 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Azure.Messaging.EventGrid" Version="4.22.0" />
+        <PackageReference Include="Azure.Messaging.EventGrid" Version="4.24.0" />
```

Comment:
Do we want to keep this as beta version or ugprade it?

Reply:
I think we should upgrade it if it's possible. In this case it ok as it doesn't use the pull delivery.